src/pages/customers/create.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../styles/common.css" />
+    <script defer src="/script/main.js"></script>
+    <title>Billbook | Create New Customer</title>
+  </head>
+
+  <body class="dashboard">
+    <nav class="nav-bar" id="nav-bar">
+      <ul>
+        <a href="#">
+          <li class="nav-item nav-item--active"><i class="material-icons">person</i> Customers</li>
+        </a>
+        <a href="/pages/items/">
+          <li class="nav-item"><i class="material-icons">star</i>Items</li>
+        </a>
+        <a href="/pages/invoice">
+          <li class="nav-item"><i class="material-icons">description</i>Invoices</li>
+        </a>
+      </ul>
```

Comment:
this is not correct. `<ul>` should always have `<li>` as child. `<a>` cannot be a the direct child of `<ul>`.
I believe this was done to have full width to be clickable, use `<a>` inside `<li>` and make `<a>` `display: block`. You can style it by doing `.nav-item > a` or any better alternative you can think of.

To validate HTML, check this out : https://validator.w3.org/nu/#textarea
Browsers are smart and unfortunately we don't get warnings for flaws in HTML, try to validate it manually.

Reply:
Have taken care of this and all the similar ones below in the latest commits.